modules/vaos/spec/services/jwt_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe VAOS::JWT do
+  subject { VAOS::JWT.new(user) }
+
+  let(:user) { build(:user, :mhv) }
+
+  describe '.new' do
+    it 'creates a VAOS::JWT instance' do
```

Comment:
This spec doesn't seem to have a real purpose. The subject is `VAOS::JWT.new(user)` -- so I don't see this being very valuable. 

Maybe remove this and inline the subject into the `#token` definition?

Reply:
This was a relic of TDD'ing the class, and I agree doesn't make much sense now, I'll remove it